src/DocumentationBrowserViewExtension/DocumentationBrowserViewExtension.cs

```diff
@@ -67,18 +68,36 @@ public override void Startup(ViewStartupParams viewStartupParams)
             {
                 var docsDir = new DirectoryInfo(Path.Combine(pathManager.DynamoCoreDirectory, FALLBACK_DOC_DIRECTORY_NAME));
                 fallbackDocPath = docsDir.Exists ? docsDir : null;
+
+                //When executing DynamoSandbox the WebView2 cache folder will be located in the same directory than the executable
+                var userDataDir = new DirectoryInfo(pathManager.DynamoCoreDirectory);
+                webBrowserUserDataFolder = userDataDir.Exists ? userDataDir : null;
             }
 
             if (!string.IsNullOrEmpty(pathManager.HostApplicationDirectory))
             {
-                var docsDir = new DirectoryInfo(Path.Combine(pathManager.HostApplicationDirectory, FALLBACK_DOC_DIRECTORY_NAME));
+                //when running over Revit HostApplicationDirectory = C:\Program Files\Autodesk\Revit 2023\AddIns\DynamoForRevit\Revit
+                //Then we need to remove the Revit folder from the path so we can find the fallback_docs directory.
+                var hostAppDirectory = Directory.GetParent(pathManager.HostApplicationDirectory).FullName;
+                var docsDir = new DirectoryInfo(Path.Combine(hostAppDirectory, FALLBACK_DOC_DIRECTORY_NAME));
                 fallbackDocPath = docsDir.Exists ? docsDir : null;
```

Comment:
So for Sandbox, this `pathManager.HostApplicationDirectory` would be null? I think this only means this is not an in process integration, for case like FormIt, where it is out of process but still delivered in programe files, this may still not work